django/db/models/options.py

```diff
@@ -90,7 +91,7 @@ def __init__(self, meta, app_label=None):
         self.unique_together = []
         self.index_together = []
         self.select_on_save = False
-        self.default_permissions = ('add', 'change', 'delete', 'view')
+        self.default_permissions = (_('add'), _('change'), _('delete'), _('view'))
```

Comment:
These strings should not be translated like this as they are used to generate the value for `Permission.codename`. This means that the values could be stored in the wrong language breaking permissions in the admin site.